src/utils.ts

```diff
@@ -0,0 +1,32 @@
+export interface PersonalNumberParts {
+  firstPart: string;
+  secondPart: string;
+}
+
+export const personalNumberAddingTwentyIssueYear = 4;
```

Comment:
asi vim co myslis, ale v kodu to nikde neni (nebo nevidim)

Reply:
https://github.com/mild-blue/personal-number/blob/2dd29536fddd185ae8b6de57752f728389dc12e3/src/validator.ts#L53